### DISCUSSION TITLE
Feature Request - Allow both page refs and hierarchy to sticky to top

### DISCUSSION THREAD

**Discussion by robbiep** [YU000jp/logseq-page-tags-and-hierarchy#17] [General]
1) It would be great if page hierarchy could be placed directly under references via a setting, as placing it at the bottom of the page requires a lot of scrolling a loss of context

Example: 
![image](https://github.com/YU000jp/logseq-page-tags-and-hierarchy/assets/5155629/1abeedbd-af32-46a7-a294-e2ce79d303a5)

2) Also, would be great to allow choosing order via settings

**Comment by YU000jp** [comment]
The current design utilizes half of the vertical space for both page tags and hierarchy. This is achieved through CSS positioning using position: fixed, which causes them to be placed regardless of the content's length. It is possible to add a configuration option to swap their positions. If you find a solution, please submit a pull request.

In the future, there is a plan to allow users the freedom to arrange and resize them by incorporating pop-up functionality.

**Reply by YU000jp** [reply]
@robbiep 
I created a new style mode.  Try `wide view` mode.
